static/js/Story.jsx

```diff
@@ -158,10 +163,11 @@ const TopicTextPassage = ({text, topic, bodyTextIsLink=false, langPref, displayD
     let innerContent = <ContentText html={{en: text.en, he: text.he}} overrideLanguage={overrideLanguage}
                                     bilingualOrder={["he", "en"]}/>;
     const content = bodyTextIsLink ? <a href={url} style={{textDecoration: 'none'}}>{innerContent}</a> : innerContent;
+    const isIntroducedSource = isCurated && displayDescription
+    const StoryFrameComp = isIntroducedSource ? SummarizedStoryFrame : StoryFrame
     return (
-        <StoryFrame
-            cls="topicPassageStory"
-            collapsibleSummary={isCurated && displayDescription ?
+        <StoryFrameComp cls="topicPassageStory"
+                        collapsibleSummary={isIntroducedSource ?
```

Comment:
this if statement isn't necessary. If it's not an introduced source, than `collapsibleSummary` won't be used (because `StoryFrame` doesn't take this prop)